server/dataGeneration/generateDump.js

```diff
@@ -2,92 +2,61 @@ const fs = require('fs');
 const path = require('path');
 const faker = require('faker');
 const command = require('./command');
+const uuid = require('uuid').v4;
 
-const iterations = 19;
+const iterations = 39;
 
 for (let k = 0; k <= iterations; k++) {
-    let query = '';
+    let authorsQuery = '';
+    let reviewsQuery = '';
 
-    let spacesQuery = '';
-    for (let i = 1 + k * 500000; i <= 500000 + k * 500000; i++) {
-        if (i === 500000 + iterations * 500000) {
-            spacesQuery += `
-workspace-${i}
-\\.
-`;
-        } else {
-            spacesQuery += `
-workspace-${i}`;
+    for (let i = 1 + k * 250000; i <= 250000 + k * 250000; i++) {
+        let first = faker.name.firstName();
+        let last = faker.name.lastName();
+        const email = faker.internet.email();
+        const hash = faker.internet.password();
+
+        while (first.indexOf("'") === 1) {
+            first = faker.name.firstName();
         }
-    }
 
-    let calculationsQuery = '';
+        while (last.indexOf("'") === 1) {
+            last = faker.name.lastName();
+        }
 
-    let reviewsQuery = '';
+        const authorId = uuid();
 
-    for (let i = 1 + k * 500000; i <= 500000 + k * 500000; i++) {
-        const calculation = {
-            id: i,
-            reviewCount: 0,
-            avg: 0,
-            total: 0,
-            spaceId: i,
-        };
+        if (i === 250000 + iterations * 250000) {
+            authorsQuery += `
+${authorId}	${first}	${last}	${email}	${hash}
+\\.`;
+        } else {
+            authorsQuery += `
+${authorId}	${first}	${last}	${email}	${hash}`;
+        }
 
         const numberOfReviews = Math.floor(Math.random() * 12);
-
         for (let j = 1; j <= numberOfReviews; j++) {
-            let first = faker.name.firstName();
-            let last = faker.name.lastName();
+            const reviewId = uuid();
             const rating = Math.floor(Math.random() * 5) + 1; // number between 1 and 5
 
-            while (first.indexOf("'") === 1) {
-                first = faker.name.firstName();
-            }
-
-            while (last.indexOf("'") === 1) {
-                last = faker.name.lastName();
-            }
-
-            calculation.reviewCount++;
-            calculation.total += rating;
-            calculation.avg = (calculation.total / calculation.reviewCount).toFixed(1);
-
-            if (j === numberOfReviews && i === 500000 + iterations * 500000) {
+            if (j === numberOfReviews && i === 250000 + iterations * 250000) {
                 reviewsQuery += `
-${first} ${last}	${faker.date
-                    .between(new Date('1970-01-01'), new Date())
-                    .toISOString()}	${rating}	${faker.lorem.sentence()}	${i}	${i}
+${reviewId}	${faker.date.between(new Date('2021-01-01'), new Date()).toISOString()}	${rating}	${faker.lorem.sentence()}	${
```

Comment:
The readability on these is odd, but I'm not familiar with Postgres conventions. Maybe break the ${Math.floor(Math.random() * 10000000) + 1 before it gets separated kind of awkwardly? Not a big deal, though, if that's the way it's supposed to be.